modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/CommentResourceImpl.java

```diff
@@ -611,22 +611,8 @@ private com.liferay.portal.kernel.comment.Comment _fetchComment(
 	private BlogsEntry _getBlogsEntry(String externalReferenceCode, Long siteId)
```

Comment:
Does it make sense to have this method that just makes a request to another service and do nothing else?